### PR TITLE
Issue 12: Update release script to adopt Trusted publisher management in pypi.

### LINKED ISSUE #12: Update release script to adopt Trusted publisher managment in pypi.
**Problem description**
Currently python release process required credentials to upload in pypi. Update release script to adopt Trusted publisher managment in pypi.

**Problem location**
Update the release script in tagPublish.yml so that it will not require user credentials for release.

**Suggestions for an improvement**
Update the release script in tagPublish.yml so that it will not require user credentials for release.

### PR DESCRIPTION
**Pull request by Bhupender-Y**
**Change log description**  
Update release script to adopt Trusted publisher management in pypi

**Purpose of the change**  
Closes #12 

**What the code does**  
Update release script to adopt Trusted publisher management in pypi

**How to verify it**  
(Optional: steps to verify that the changes are effective)

### CODE REVIEW

**Review by AnkurAnand11 — APPROVED** [review]
LGTM

File: .github/workflows/tagPublish.yml [hunk 1]
```diff
@@ -66,11 +69,28 @@ jobs:
       - name: build source
         id: build_source
         run: python -u .github/workflows/build_source.py
+      - name: mint API token
+        id: mint-token
+        shell: bash
+        run: |
+            # retrieve the ambient OIDC token
```
Comment (tkaitchuck) [inline .github/workflows/tagPublish.yml:77]: Add a comment linking to where this documentation was found

**Review by tkaitchuck — APPROVED** [review]
Otherwise looks good

File: .github/workflows/tagPublish.yml (hunk 1 above)
Comment (Bhupender-Y) [reply .github/workflows/tagPublish.yml:77]: Done

**Review by ShwethaSNayak — APPROVED** [review]
LGTM

**Pull request merged**